iron-list.html

```diff
@@ -756,10 +756,12 @@
      * if the physical size is shorter than `_optPhysicalSize`
      */
     _increasePoolIfNeeded: function() {
-      if (this._viewportSize !== 0 && this._physicalSize < this._optPhysicalSize) {
-        // 0 <= `currentPage` <= `_maxPages`
-        var currentPage = Math.floor(this._physicalSize / this._viewportSize);
+      if (this._viewportSize !== 0 &&
+          this._physicalSize < this._optPhysicalSize &&
+          this._virtualEnd < this._virtualCount-1) {

```

Comment:
This issue happens when the `_physicalSize` is smaller than `_optPhysicalSize` and the user calls `scrollToIndex` to one of last indexes. 

`scrollToIndex` does `this._virtualStart = idx - 1;` and that would result in `_physicalSize` never reaching `_optPhysicalSize`.

 https://github.com/PolymerElements/iron-list/blob/master/iron-list.html#L1155